documentation/src/main/asciidoc/topics/code_examples/AnalyzerDefinitions.java

```diff
@@ -1,7 +1,7 @@
-@ProtoDoc("@Field(store = Store.YES, analyze = Analyze.YES, analyzer = @Analyzer(definition = \"keyword\"))")
+"@Field(store = Store.YES, analyze = Analyze.YES, analyzer = @Analyzer(definition = \"keyword\"))
```

Comment:
I think the quotes aren't required anymore, right? neither the backslash `definition = \"keyword\"`

Reply:
same below